### ISSUE TITLE
Slimefun items trade blocked on custom Shopkeepers plugin villagers

### ISSUE THREAD

**Issue by andris155** [Slimefun/Slimefun4#2581]
## :round_pushpin: Description (REQUIRED)

Slimefun items trade blocked on custom Shopkeepers plugin villagers

Shopkeepers plugin:
Set up custom villager shopkeepers that sell exactly what you want them to sell and for what price. You can set up admin shops, which have infinite supply, and you can also set up player shops, which pull supply from a chest.

The Slimefun currently block Slimefun items trade on this villagers, but players want to set up villagers for themselves where they can trade Slimefun items.

## :bookmark_tabs: Steps to reproduce the Issue (REQUIRED)

1. Create custom villager this trade: Steel Ingot -> Oak Log
2. Try trade with villager.
3. [CHAT] Slimefun 4> You cannot trade Slimefun items with Villagers!

## :bulb: Expected behavior (REQUIRED)

Slimefun items trade enabled on custom Shopkeepers plugin villagers

## :open_file_folder: /error-reports/ Folder

No error report.

## :compass: Environment (REQUIRED)

[17:27:39 INFO]: This Server uses the following setup of Slimefun:
[17:27:39 INFO]: Yatopia v1_16_R3
[17:27:39 INFO]: CS-CoreLib vDEV - 91 (git e42a05d1)
[17:27:39 INFO]: Slimefun vDEV - 757 (git 77f4373b)
[17:27:39 INFO]: Metrics build: #17
[17:27:39 INFO]:
[17:27:39 INFO]: Installed Addons: (1)
[17:27:39 INFO]:  LiteXpansion vDEV - 98 (git f79eab2f)

**Comment by TheBusyBiscuit** [comment]
As the message says "You cannot use Slimefun items to trade with villagers".
Removing this would open up a lot of unintended side effects and re-allow lots of bugs.
Also, this doesn't constitute a bug anyway, Slimefun works as intended - it blocks you from using Slimefun items for trading.

If anything, this would be more of a suggestion and should be posted on discord instead.
Anyway, not a bug, works as intended.

**Comment by TheBusyBot** [comment]
Your issue has been closed by an admin, it may fall under one or more of the following categories.
**Please wait for an admin to tick off the points that apply.**
<hr>

* [ ] You did not follow our template. Please follow the Issue template to help us identify your issue more effectively.
* [ ] You did not provide any information about your versions (We absolutely need the exact version numbers that are installed on your Server, \"latest\" is not helpful)
* [ ] You did not provide a proper description to the problem. Try to write at least 4-6 sentences and/or provide screenshots or videos on how to reproduce this.
* [ ] We were unable to reproduce issue, if you think your issue still persists then please comment down below and give a better description on how to reproduce it.
* [ ] Your issue was posted in a foreign language, we only accept english issues on here.
* [ ] Your issue is not a bug, it is intended to work this way.
* [ ] Your issue is not really a bug, it is a limitation or simply something we have no control over.
* [ ] Your issue is not a bug, please only use this issue tracker to report bugs. Any other kind of communication should happen on discord.
* [ ] Your issue has already been reported before, it is a duplicate. Check the other issues first before posting!
* [ ] You posted an error without using pastebin. Please always post errors via pastebin otherwise they become nearly unreadable.
* [ ] You seem to be reporting multiple bugs at once. Please make a separate issue for each bug you encountered, so we can properly handle them individually.
* [ ] Your issue has already been fixed in a later version of Slimefun or CS-CoreLib, you should update.
* [ ] You are using an outdated and unsupported version of Slimefun / CS-CoreLib, again, you should update.
* [ ] You are using an unofficially modified build of Slimefun. We only support official versions of Slimefun - for obvious reasons.
* [ ] You are using an unsupported version of Minecraft. We only provide support for the Minecraft versions Slimefun was developed for, older versions are not supported anymore.
* [ ] You are using a \"stable\" version of Slimefun (prefixed with \"RC - \"), your issue may have been fixed in a development build, so we only accept bug reports from those.
* [ ] You are on the wrong issue tracker. We would like to remind you that this Issue Tracker is **only for Slimefun**. To report bugs on any addons, head to the corresponding issue tracker of that addon.
<hr>

Please respond below, if you have any further questions.
Do **not** open a new Issue unless explicitly told otherwise, comment below or edit your post instead.<br>
Make sure to check out our article on [How to report bugs](https://github.com/Slimefun/Slimefun4/wiki/How-to-report-bugs) for even more information.
